<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>285</width>
    <height>428</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
background: white;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnComprobante">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
color: #7855F6;
border: 1px solid #786DA0;
border-radius: 10px
}

QPushButton:hover {
color: white;
background: #7855F6;
border: 1px solid white;
};</string>
    </property>
    <property name="text">
     <string>Comprobante de Pago</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>241</width>
      <height>4</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #786DA0;
min-height: 2px;
max-height: 2px;</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btnProductos">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
color: #7855F6;
border: 1px solid #786DA0;
border-radius: 10px
}

QPushButton:hover {
color: white;
background: #7855F6;
border: 1px solid white;
};</string>
    </property>
    <property name="text">
     <string>Registro de Productos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnEmpleados">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
color: #7855F6;
border: 1px solid #786DA0;
border-radius: 10px
}

QPushButton:hover {
color: white;
background: #7855F6;
border: 1px solid white;
};</string>
    </property>
    <property name="text">
     <string>Registro de Empleados</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSalir">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>300</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#btnSalir {
background: #FA907D;
color: black;
border-radius: 10px;
}

QPushButton#btnSalir:hover {
background-color: #FA4E42;
}</string>
    </property>
    <property name="text">
     <string>Cerrar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #7855F6;
font: tahoma;
font-style: bold;
font-size: 16pt;</string>
    </property>
    <property name="text">
     <string>MENU PRINCIPAL</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnClientes">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
color: #7855F6;
border: 1px solid #786DA0;
border-radius: 10px
}

QPushButton:hover {
color: white;
background: #7855F6;
border: 1px solid white;
};</string>
    </property>
    <property name="text">
     <string>Registro de Clientes</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>350</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #7855F6;
</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span&gt;Fernando Ruiz&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>285</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
